packages/shared/src/contexts/SettingsContext.tsx

```diff
@@ -192,7 +192,11 @@ export const SettingsContextProvider = ({
       setSpaciness: (density) =>
         setSettings({ ...settings, spaciness: density }),
       toggleInsaneMode: () =>
-        setSettings({ ...settings, insaneMode: !settings.insaneMode }),
+        setSettings({
+          ...settings,
+          insaneMode: !settings.insaneMode,
+          spaciness: !settings.insaneMode ? 'eco' : settings.spaciness,
```

Comment:
How about existing users on list mode now?

Reply:
nah we don't tackle it for existing users unless they switch. As discussed in chat, let's not do this change first just disable the options for users when in list mode assuming the issues have been a one off. And then we can consider adding this after if users complain again.